test/API/endpoints/index.spec.js

```diff
@@ -0,0 +1,149 @@
+    assert.deepEqual(ret.path, '/meetings/12345');
+    assert.deepEqual(ret.method, 'PATCH');
+    assert.deepEqual(ret.params, { stubParam: '1' });
+  });
+
+  it('updates Meeting with occurrenceId present', async function () {
+    const ret = await testAPI.meeting.update({ meetingId: 12345, meetingObj: {}, occurrenceId: 'present' });
+    assert.deepEqual(ret.path, '/meetings/12345?occurrence_id=present');
+    assert.deepEqual(ret.method, 'PATCH');
+    assert.deepEqual(ret.params, {});
+  });
+
+  it('deletes Meeting', async function () {
+    const ret = await testAPI.meeting.delete({ meetingId: 12345 });
+    assert.deepEqual(ret.path, '/meetings/12345');
+    assert.deepEqual(ret.method, 'DELETE');
+    assert.deepEqual(ret.params, { schedule_for_reminder: false });
+  });
+
+  it('deletes Meeting with optional params', async function () {
+    const ret = await testAPI.meeting.delete({ meetingId: 12345, notifyHosts: 'yes', occurrenceId: 55 });
+    assert.deepEqual(ret.path, '/meetings/12345');
+    assert.deepEqual(ret.method, 'DELETE');
+    assert.deepEqual(ret.params, { schedule_for_reminder: true, occurrence_id: '55' });
+  });
+
+  it('lists Past Meeting Instances', async function () {
+    const ret = await testAPI.meeting.listPastInstances({ meetingId: 12345 });
+    assert.deepEqual(ret.path, '/past_meetings/12345/instances');
+    assert.deepEqual(ret.method, 'GET');
+    assert.deepEqual(ret.params, undefined);
+  });
+});
+
+// TODO: Test without post-processor altering response object
+describe('User Endpoints', async function () {
+  it('gets ZAKToken', async function () {
+    const ret = await testAPI.user.getZAKToken({ userId: 12345 });
+    // Post processor makes response object just the token which would
+    // be undefined for the fake user
+    assert.strictEqual(ret, undefined);
+  });
+});
+
+describe('Cloud Recording Endpoints', async function () {
+  it('lists Meeting Recordings', async function () {
+    const ret = await testAPI.cloudRecording.listMeetingRecordings({ meetingId: 12345 });
+    assert.deepEqual(ret.path, '/meetings/12345/recordings');
+    assert.deepEqual(ret.method, 'GET');
+    assert.deepEqual(ret.params, undefined);
+  });
+
+  it('lists Meeting Recordings with double encoding needed (Test 1)', async function () {
+    const ret = await testAPI.cloudRecording.listMeetingRecordings({ meetingId: '/12345' });
+    assert.deepEqual(ret.path, '/meetings/%252F12345/recordings');
+    assert.deepEqual(ret.method, 'GET');
+    assert.deepEqual(ret.params, undefined);
+  });
+
+  it('lists Meeting Recordings with double encoding needed (Test 2)', async function () {
+    const ret = await testAPI.cloudRecording.listMeetingRecordings({ meetingId: '123//45' });
+    assert.deepEqual(ret.path, '/meetings/123%252F%252F45/recordings');
+    assert.deepEqual(ret.method, 'GET');
+    assert.deepEqual(ret.params, undefined);
+  });
```

Comment:
General comment about tests: your coverage is great!
...but please document your tests just like you would document code

```
// Call the stub API
...

// Make sure the correct request body was sent to Zoom
...
```